R/colors.R

```diff
@@ -146,3 +146,57 @@ scale_fill_gouv_continuous <- function(
 
   ggplot2::scale_fill_gradientn(colors = pal(256), ...)
 }
+
+#' Display palettes used in French governmental documents
+#'
+#' Plot the palettes used in `gouvdown`.
+#'
+#' @inheritParams gouv_pal_inter
+#'
+#' @export
+#'
+#' @examples
+#' display_palette("pal_gouv_fr")
+#' display_palette_all()
+display_palette <- function(palette) {
```

Comment:
from https://github.com/spyrales/gouvdown/pull/13#issuecomment-620628478

```suggestion
display_palette <- function(palette = "scale_color_gouv_discrete") {
```